website/source/docs/platform/k8s/ambassador.html.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: "docs"
+page_title: "Ambassador Integration - Kubernetes"
+sidebar_current: "docs-platform-k8s-ambassador"
+description: |-
+    Ambassador is a Kubernetes-native API gateway and ingress controller that 
+    integrates well with Consul Connect.
+---
+
+# Ambassador Integration with Consul Connect
+
+Besides enabling Kubernetes services to discover and securely connect to each other,
+Connect also can help route traffic into the cluster from outside, when paired with 
+an [ingress controller] like DataWire's Ambassador.
+ 
+[Ambassador] is a popular Kubernetes-native service that acts as an ingress controller
+or API gateway. It supports an optional integration with Consul that allows it to
+route incoming traffic to the [proxies] for your Connect-enabled services. 
+
+This means you can have **end-to-end encryption** from the browser, to Ambassador,
+to your Kubernetes services. 
+
+
+## Installation 
+
+First, install Consul and enable Connect on the agents inside the cluster. Decide
```

Comment:
"install consul" and "enable connect" should link out to relevant docs/guides

Reply:
Fixed with 9d2a65bda0c87f988c71ed09f0ccc8b7b11b51c3.